packages/sshnoports/tools/Dockerfile

```diff
@@ -1,3 +1,6 @@
+# Dockerfile
+# Build image for a containerized instance of sshnpd
+
```

Comment:
Labelling all of our dockerfiles so we know what they do and why they need to exist